Marsiple/App/Todo Form/TodoFormViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  TodoFormViewController.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 31/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class TodoFormViewController: UIViewController {
+    // NOTE: id is set to -1 so we can detect if we're creating new or changing existing todo (to detect wich initializer is used)
+    private var todo = Todo(id: -1, title: "", completed: false, userId: 1)
+    private let todoFormView = TodoFormView.autolayoutView()
+    private lazy var completedButton: UIButton = {
+        let button = UIButton()
+        let image = todo.completed ? #imageLiteral(resourceName: "checkbox-selected") : #imageLiteral(resourceName: "checkbox-unselected")
+        button.setImage(image, for: .normal)
+        button.addTarget(self, action: #selector(completedButtonTapped), for: .touchDown)
+        button.snp.makeConstraints {
+            $0.width.height.equalTo(30)
+        }
+        return button
+    }()
+    
+    init() {
+        super.init(nibName: nil, bundle: nil)
+        setupView()
+    }
+    
+    init(todo: Todo) {
+        self.todo = todo
+        super.init(nibName: nil, bundle: nil)
+        setupView()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+private extension TodoFormViewController {
+    @objc func completedButtonTapped() {
+        todo.completed = !todo.completed
+        let image = todo.completed ? #imageLiteral(resourceName: "checkbox-selected") : #imageLiteral(resourceName: "checkbox-unselected")
```

Comment:
With the previously mentioned code (with the different button images depending on the button's state), we could just use `completedButton.isSelected = todo.completed` here.